lib/ArduinoNative/src/WiFiUdp.cpp

```diff
@@ -207,9 +268,28 @@ void WiFiUDP::println(int32_t value)
 
 size_t WiFiUDP::readBytes(uint8_t* buffer, size_t length)
 {
-    UTIL_NOT_USED(buffer);
-    UTIL_NOT_USED(length);
-    return 0;
+    if (m_inBufferLength == 0)
```

Comment:
Should also protect agains zero byte reads.

Reply:
Aborting if zero bytes are requested or no bytes are available to read now.